src/plrust.rs

```diff
@@ -43,6 +43,23 @@ pub(crate) unsafe fn unload_function(fn_oid: pg_sys::Oid) {
     })
 }
 
+#[tracing::instrument(level = "debug")]
+pub(crate) unsafe fn recompile_missing_shared_object(fn_oid: pg_sys::Oid) -> eyre::Result<()> {
+    let db_oid = unsafe { MyDatabaseId };
```

Comment:
This repeats the function logic inside `evaluate_function`, thus manages to dodge safety comments on this.